.travis.yml

```diff
@@ -51,6 +51,7 @@ matrix:
     - php: '7.3'
       env: SYMFONY_DEPRECATIONS_HELPER=0
   allow_failures:
+    - php: 7.4snapshot
```

Comment:
Why did you add this line? There is no such job, is there?

Reply:
wrong line added..fixed